src/components/Root.tsx

```diff
@@ -0,0 +1,23 @@
+import { HashRouter, Navigate, Route, Routes } from "react-router-dom"
+import { ErrorPage } from "./ErrorPage"
+import { PeoplePage } from "./PeoplePage"
+import { App } from "../App"
+import { HomePage } from "./HomePage"
+
+export const Root = () => {
+  return (
+    <HashRouter>
+      <Routes>
+      <Route path='/' element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path="home" element={<Navigate to="/" replace />}/>
+        <Route path="people">
+          <Route index element={<PeoplePage />} />
+          <Route path=":personSlug" element={<PeoplePage />} />
+        </Route>
+        <Route path="*" element={<ErrorPage />} />
+      </Route>
```

Comment:
The way routes are nested might cause some issues. Currently, all routes are nested inside the `/` route, which means the `App` component will always be rendered, regardless of the path. If the `App` component doesn't need to be always present, you might want to consider revising the routing structure.